Makefile.build

```diff
@@ -735,7 +735,8 @@ $(MICROMEGAV:.v=.vo) $(MICROMEGAV:.v=.glob) : $(MICROMEGAV) theories/Init/Prelud
 	$(SHOW)'COQC      $<'
 	$(HIDE)rm -f $*.glob
 	$(HIDE)$(BOOTCOQC) $< | sed -e '$$d' > $(MICROMEGAGEN)
-	$(HIDE)diff -u --strip-trailing-cr $(MICROMEGAML) $(MICROMEGAGEN) || \
+        # Alpine's Linux diff is quite restricted, hence the -b -B
+	$(HIDE)diff -u -b -B $(MICROMEGAML) $(MICROMEGAGEN) || \
```

Comment:
Your comment about Alpine is rather ill-informed. 

This is the part of diff that you are allowed to use if you want to be portable: http://pubs.opengroup.org/onlinepubs/9699919799/utilities/diff.html. As you can see, you failed.

Having operating specific hacks leads to non-portable hacks and a mess (that's your current path).

Reply:
Coq has no interest on POSIX portability, which is basically a fairy tale.

Reply:
Why not? Because you don't know how?